RecDP/tests/test_llmutils.py

```diff
@@ -74,22 +74,14 @@ def test_text_to_jsonl(self):
         text_to_jsonl_MP(data_dir, out_dir, 2)
 
     def test_ppi_remove(self):
-        from dataclasses import dataclass
+        from pyspark.sql import SparkSession
 
-        @dataclass
-        class PiiDetectRedactOption:
-            path: str = "json"
-            data_files: str = "tests/data/llm_data/arxiv_sample_100.jsonl"
-            split: str = "train"
-            text_column: str = "text"
-            batch_size: int = 100
-            num_proc: int = 8
-            seed: int = 10
-            save_path: str = "./pii_remove"
-            save_format: str = "json"
+        spark = SparkSession.builder.master("local[2]").appName(
```

Comment:
please use pyrecdp embedded way to start spark

Reply:
@xuechendi what are the reasons for using this? I think this Spark data processor is a bit complex if only use for unit test?

Reply:
@chaojun-zhang, the goal is to align with all other provided components with same API to enable spark.
if we are seeing issues, we don't have multiple branches. Please update to use the other interface.

Meanwhile, SparkDataProcess provides other materialization options.